impeller/aiks/canvas.cc

```diff
@@ -39,7 +39,10 @@ static bool UseColorSourceContents(
   return !vertices->HasTextureCoordinates();
 }
 
-Canvas::Canvas() {
+Canvas::Canvas()
+    : checkerboard_color_proc_([]() {
```

Comment:
This is perhaps too configurable? Can we just make up this pattern ourselves? Or make this an implementation detail?

Reply:
I should clarify my thinking here. I was hoping to make the checkerboarding more useful by perhaps having nested save layers be rendered differently (via progressively darkening overlays maybe).

Reply:
Making the color non-configurable/just based on depth SGTM, but what about textures at the same depth? Maybe we can kick that can down the road, but it might make sense to make the edges of checkerboarded textures look distinct by rendering a solid border a few pixels wide.

Reply:
Also, should color remain consistent between frames?

It seems that hue should vary within a level, but darkness or opacity changes as you increase the level?

Reply:
I made the checkerboards red with 0.25 alpha + get progressively darker based on depth. At level >= 5 they're just black. I can add a neat stable way to identify passes in a follow-up and hue shift based on it -- this kind of thing will be useful for other tooling as well, and we should make it deterministic for multithreaded encoding.